app/build.gradle

```diff
@@ -19,13 +19,23 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.google.firebase.crashlytics'
 apply from: 'jacoco.gradle'
 
+def githubProperties = new Properties()
+githubProperties.load(new FileInputStream(rootProject.file("github.properties")))
+
 repositories {
     // for local aar inclusion
     flatDir {
         dirs 'libs'
     }
     maven { url "https://jitpack.io" }
-    maven { url "http://dl.bintray.com/ona/kujaku" }
+    maven {
+        name = "GitHubPackages"
+        url = uri("https://maven.pkg.github.com/onaio/kujaku")
+        credentials {
+            username = githubProperties['gpr.usr'] ?: System.getenv("GPR_USER")
```

Comment:
`System.getenv("GPR_USER")` is getting value from jenkins job config right? 
If yes, Should we rather use `gradle.properties` here, since creating new jobs in jenkins would require adding this property.

Reply:
good point, will update.